gammapy/spectrum/spectral_grouping.py

```diff
@@ -0,0 +1,133 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import logging
+from astropy.coordinates import Angle
+from astropy.units import Quantity
+import numpy as np
+from ..spectrum.spectrum_extraction import SpectrumObservationList, SpectrumObservation
+from ..data import ObservationGroupAxis, ObservationGroups
+
+__all__ = [
```

Comment:
Try: ``make clean``
or ``python -c 'import gammapy.spectrum'``
to see if there really is a SyntaxError in that file.